hugr/src/lib.rs

```diff
@@ -140,7 +140,6 @@
 // https://github.com/proptest-rs/proptest/issues/447
 #![cfg_attr(test, allow(non_local_definitions))]
 
-pub mod algorithm;
```

Comment:
Once we do #294 we should re-export algorithms here.

Normally we sub-crates are used for specific components of the library and the main library re-exports all non-internal things, but that will have to wait for now.